canonical/lxd-ui
# feat: [WD-15076] Instance Overview add devices

## Done
Work Done:
- Added Pool to Instance Overview page
- Added Pool to Instance Detail Panel Content
- Added PID to Instance Detail Panel Content
- Added Devices to Instance Overview Page

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Navigate to an Instance and observe the changes in it's Overview and side panel.

## Screenshots

![image](https://github.com/user-attachments/assets/81dd6a60-0d16-4038-a5d1-ed121c641f54)
